.github/workflows/docker-health-check.yml

```
@@ -4,44 +4,91 @@ on:
+  pull_request:
+    branches: 
+      - main
+      - 'fix/*'
 
 jobs:
   health-check:
     runs-on: ubuntu-latest
     
     steps:
-    - name: 코드 체크아웃
+    - name: Checkout Code
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)